FutureNorthants/northants-website
# Added settings to tweak Solr search, removed negative boost feature

### Testing
1. Checkout this branch and run:
```
./drush.wrapper @docker updb
./drush.wrapper @docker cim
./drush.wrapper @docker cache-rebuild
```
In the CMS, verify that you now have a new menu entry under Configuration | Search and metadata, namely **Solr search tweaks**. Go into this new screen and tick all the boxes if they are not ticked already and save. Verify they all remain ticked after saving.
Now rebuild the news and main search indexes in the backend.
2. Now in the frontend, search for 'electoral services'. Verify that the associated privacy policy page does not appear.
3. Untick the 'Exclude privacy notices' setting in the backend and retry your search in the frontend. Verify the privacy policy page now appears in the results.
4. Play with the other two settings and verify they have an effect on search results.
5. Repeat for the news articles page searching.

The intention is, if no faults are found in the PR, this can go into staging or a new dev instance for Richard & co to experiment with the tweaks to see if there's a combination they like.
